.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
```

Comment:
```markdown
- The concatenation seems incorrect here. `RELEASE_NOTES` should be updated with `new_fix_notes` instead of itself.
- The order of variables in `RELEASE_NOTES` might need to be reviewed for consistency.
- The variable `updated_content` is being assigned with `existing_content` which is not modified here.
```